inc/user.class.php

```diff
@@ -3020,8 +3020,7 @@ static function getSqlSearchResult ($count=true, $right="all", $entity_restrict=
 
                   case 'create_ticket_validate' :
                      $where[]= " (`glpi_profilerights`.`name` = 'ticketvalidation'
-                                  AND (`glpi_profilerights`.`rights` & ".TicketValidation::CREATEREQUEST."
-                                       OR `glpi_profilerights`.`rights` & ".TicketValidation::CREATEDEMAND.") ".
```

Comment:
Imo, we should replacing TicketValidation::CREATEDEMAND by TicketValidation::CREATEINCIDENT
instead removing line